### PR TITLE
linear search in rust

### LINKED ISSUE #2179: Linear Search
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [x] Dart - @keshakaneria
- [x] Go
- [x] Java
- [x] Julia
- [x] JS
- [x] Kotlin
- [x] Php
- [x] Python
- [x] Readme
- [x] Ruby
- [x] Rust - @raksha009
- [x] TypeScript - @mruna7

### PR DESCRIPTION
**Pull request by raksha009**
linear search in rust

Fixes #2179 

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [X] Input is taken dynamically.
- [X] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added new file
- 

#### Languages Used:
- Rust
- 

#### Files Added:
- Linear_Search.rs
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by raksha009** [comment]
plz review my pr @singh-shreya6 @asha15 maam

File: Linear_Search/Linear_Search.rs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        vector.push(temp_arr);
+    }
+    
+    // Linear Search process
+    let mut checkpoint = 0;
+    for index in 0..size - 1 {
+        if element == vector[index]{
+            checkpoint = 1;
+            println!("Element is found at index ");
+    		println!("{:?}" , index + 1);
```
Comment (singh-shreya6) [inline Linear_Search/Linear_Search.rs:34]: Incorrect indentation.

File: Linear_Search/Linear_Search.rs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        }
+    }
+    
+    if  checkpoint == 0 {
```
Comment (singh-shreya6) [inline Linear_Search/Linear_Search.rs:38]: Extra space between if and checkpoint

**Comment by raksha009** [comment]
done the changes @singh-shreya6 maam

**Pull request merged**